### ISSUE TITLE
I can no longer see my Arizen wallet- please help

### ISSUE THREAD

**Issue by bobklein58** [ZencashOfficial/arizen#269]
I set up a new wallet a few weeks ago and deposited zencash in it.  When I go into my Arizen account today that wallet no longer shows up.  I have hard copy of my wallet # and my private key # but can not seem to be able to find a way to restore.  Please help

**Comment by gsfakianakis** [comment]
Hello,

what do you mean not show up ? Can you see the login screen and login ?

**Comment by bobklein58** [comment]
Thank you for replying.  When I log into Arizen, the wallet I originally had zencash in no longer shows up.  When I click onshow accounts with 0 balances they are all empty.  I know the wallet address but do not know how to get it to show up again.  Does that clarify your question?  My login seems to work but can not see my specific wallet with the balance in it.

**Comment by gsfakianakis** [comment]
Do you see addresses with 0 balance ? 
OR
no addresses at all ?

**Comment by bobklein58** [comment]
I can see multiple wallets with 0 balances when I hit button that say show 0 balances.

Sent from my iPhone

On Jan 30, 2018, at 4:01 PM, gsfakianakis <notifications@github.com<mailto:notifications@github.com>> wrote:


Do you see addresses with 0 balance ?
OR
no addresses at all ?

**Comment by bobklein58** [comment]
I cannot see the address where my zencash is held.  How do I get to that address?  I appreciate your willingness to help.  It is very frustrating without any guidance or someone to talk with.

**Comment by bobklein58** [comment]
I am still unable to understand how to re-establish an address.  I have the wallet address and the private key address printed on a hard copy.  I have my user name and password that are still working.  What am I missing??

From: Bob Kleinschmidt [mailto:bklein58@outlook.com]
Sent: Tuesday, January 30, 2018 4:43 PM
To: Bob Kleinschmidt <bklein58@outlook.com>; ZencashOfficial/arizen <reply@reply.github.com>
Subject: RE: [ZencashOfficial/arizen] I can no longer see my Arizen wallet- please help (#269)

I cannot see the address where my zencash is held.  How do I get to that address?  I appreciate your willingness to help.  It is very frustrating without any guidance or someone to talk with.

**Comment by bobklein58** [comment]
I was finally able to find the AWD file in an windows.old file and imported it back into my account.  I can sleep at night now!   The hard copies of the wallet and private keys never really helped in anyway.

From: Bob Kleinschmidt [mailto:bklein58@outlook.com]
Sent: Thursday, February 1, 2018 4:28 PM
To: ZencashOfficial/arizen <reply@reply.github.com>
Cc: Bob Kleinschmidt <bklein58@outlook.com>
Subject: RE: [ZencashOfficial/arizen] I can no longer see my Arizen wallet- please help (#269)

I am still unable to understand how to re-establish an address.  I have the wallet address and the private key address printed on a hard copy.  I have my user name and password that are still working.  What am I missing??

From: Bob Kleinschmidt [mailto:bklein58@outlook.com]
Sent: Tuesday, January 30, 2018 4:43 PM
To: Bob Kleinschmidt <bklein58@outlook.com<mailto:bklein58@outlook.com>>; ZencashOfficial/arizen <reply@reply.github.com>
Subject: RE: [ZencashOfficial/arizen] I can no longer see my Arizen wallet- please help (#269)

I cannot see the address where my zencash is held.  How do I get to that address?  I appreciate your willingness to help.  It is very frustrating without any guidance or someone to talk with.

**Comment by gsfakianakis** [comment]
Sorry for the late response.

You most probably created a new address and you had a diferent back up file.

**Keep you backed up keys.**
In version 1.1.4 there will be option to **import single private key**.

**Comment by bobklein58** [comment]
My windows system had sent a message it was having problems and asked to be reset.  The reset caused me to lose the backup file.   It was stored in an windows.old location.   I did not realize I could not type in my address from hard copy.   I had retained all of the details via hard copy.  I did not know the address was in an .awd file.     When I tried opening the .awd file in windows it kept saying I did not have the correct program.  I was finally able to copy it in an email to self and then imported into Arizen by cut and paste from email attachment.  Thank you for the follow up.
